.changeset/eighty-crews-rhyme.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core': minor
```

Comment:
Should be a `patch` change right? Nothing breaking here

Reply:
:+1: Yup, I'll swap that to patch and update the branch. This is good to go as is then.